devops/developer-api/openapi.yml

```diff
@@ -0,0 +1,1843 @@
+      summary: Set Index Plan
+      description: This endpoint is used to change the plan of an index.
+      x-mint:
+        href: /devops/developer-api/vector/set_index_plan
+      operationId: setIndexPlan
+      parameters:
+        - name: id
+          in: path
+          description: The unique ID of the index to change plan for
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                target_plan:
+                  type: string
+                  description: The new plan for the index
+                  enum:
+                    - payg
+                    - fixed
+              required:
+                - target_plan
+      responses:
+        "200":
+          description: Index plan changed successfully
+          content:
+            application/json:
+              schema:
+                type: string
+                example: "OK"
+      security:
+        - basicAuth: []
+  /vector/index/{id}/transfer:
+    post:
+      tags:
+        - vector
+      summary: Transfer Index
+      description: This endpoint is used to transfer an index to another team.
+      x-mint:
+        href: /devops/developer-api/vector/transfer_index
+      operationId: transferIndex
+      parameters:
+        - name: id
+          in: path
+          description: The unique ID of the index to be transferred
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                target_account:
+                  type: string
+                  description: The ID of the target account. If the target is a team, then use the format `team:<TEAM_ID>`, else if the target is your personal account use the format `<YOUR_EMAIL>`.
+                  example: "team:team-id-1"
+              required:
+                - target_account
+      responses:
+        "200":
+          description: Index transferred successfully
+          content:
+            application/json:
+              schema:
+                type: string
+                example: "OK"
+      security:
+        - basicAuth: []
+components:
+  schemas:
+    Database:
+      type: object
+      properties:
+        database_id:
+          type: string
+          description: ID of the database
+          example: 96ad0856-03b1-4ee7-9666-e81abd0349e1
+        database_name:
+          type: string
+          description: Name of the database
+          example: MyRedis
+        database_type:
+          type: string
+          description: Type of the database in terms of pricing model
+          example: Pay as You Go
+          enum:
+            - Free
+            - Pay as You Go
+            - Enterprise
```

Comment:
I am guessing this isn't true

I remember that you mentioned the API being out of date (it probably is).

Can you go over the types once again? While going over them, it would be great if you can verify against the API or check the types in console